docs/3.api/4.commands/upgrade.md

```diff
@@ -31,7 +31,7 @@ Option | Default | Description
 --- | --- | ---
 `--cwd=<directory>` |  | Specify the working directory, this takes precedence over ROOTDIR (default: `.`)
 `--logLevel=<silent\|info\|verbose>` |  | Specify build-time log level
-`--dedupe` |  | Will deduplicate dependencies but not recreate the lockfile
+`--dedupe` |  | Dedupe dependencies after upgrading
```

Comment:
This is the description used in nuxt/cli, the original description is not entirely accurate as was mentioned here https://github.com/nuxt/nuxt/pull/31467#discussion_r2005322280